common/persistence/cassandraPersistence.go

```diff
@@ -1617,7 +1617,7 @@ func (d *cassandraPersistence) GetTransferTasks(request *GetTransferTasksRequest
 		defaultVisibilityTimestamp,
 		request.ReadLevel,
 		request.MaxReadLevel,
-		request.BatchSize)
+	).PageSize(request.BatchSize).PageState(request.NextPageToken)
```

Comment:
How do you guarantee parameters for the query are not changed between multiple calls for different pages?